os/pm/Kconfig

```diff
@@ -42,6 +42,45 @@ config PM_METRICS_DURATION
 
 endif
 
+comment "IDLE PM Options"
+
+config IDLE_PM
+	bool "TIZENRT IDLE PM framework"
+	default n
+	depends on ARCH_HAVE_IDLE_PM
+	---help---
+		Used to enable TizenRT IDLE PM framework
+
+config ARCH_IDLE_PM_STATE_ENTER_THRESH
```

Comment:
I guess this also should be part of dynamic input.. 
If SOC supports 2 states
STANDBY and STOP
They can decide what should be the ACTIVITY THRESHOLD LEVEL to enter to different states

So this config should be declared something similar to  ARCH_IDLE_PM_STATE_COUNT_THRESH 

where, default value can be NULL
if default value is NULL => we can use 1 as default value in code
if SOC wants, they can give different values like "1,2"

Hint:
The way you have handled the different sleep states, same way  activity level for different sleep states should be handled